.github/workflows/auto-review-trigger.yml

```diff
@@ -38,6 +41,12 @@ jobs:
         if: github.event_name == 'workflow_dispatch'
         env:
           PR_NUMBER: ${{ inputs.pr_number }}
+
+      - name: Write PR Number - Comment Retrigger
+        run: echo $PR_NUMBER > pr-number.txt
+        if: github.event_name == 'issue_comment' && github.event.issue.pull_request && startsWith(github.event.issue.comment.body, '@eth-bot retrigger')
```

Comment:
Suuuper annoying nits, but could we go with contains instead of `startsWith`, and maybe `@eth-bot rerun`?

Reply:
Fine.